web/libs/js/main.dash2.js

```diff
@@ -2046,7 +2046,7 @@ $.timelapse.e.on('click','[timelapse]',function(){
                     video=e.video
```

Comment:
I'm not totally sure?  The red highlights are supposed to be GitHub's syntax checking, but I can't see anything wrong and the code works fine without errors...